CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [unreleased]
 
+- Features: Add [Jest](https://facebook.github.io/jest/) as our new
+  testing framework instead of [Mocha](https://mochajs.org/).
```

Comment:
J'ajouterai `instead of Mocha and Chai`. Mocha, étant la lib qui permet de lancer les tests et Chai, la librairie d'assertion.